CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
-### Latest Release (1.14.3 - May 3, 2021)
+### Latest Release (1.14.4 - June 11, 2021)
```

Comment:
We should link the related milestone for v.1.14.4 in these release notes (https://github.com/awslabs/amazon-kinesis-client/milestone/61)), and also tag any PR's (#816) being merged as part of this release under this milestone.

Reply:
Done